From 4933437c30c7fc359278548fdcac07ba422a4453 Mon Sep 17 00:00:00 2001
From: Robin Gong <b38343@freescale.com>
Date: Wed, 30 Sep 2015 16:07:06 +0800
Subject: [PATCH] MLK-11660: spi: spi-imx: fix building error

fix building error by d5150a37c14:

drivers/spi/spi-imx.c: In function 'spi_imx_can_dma':
drivers/spi/spi-imx.c:211:3: error: expected ')' before 'return'
   return true;
      ^
      drivers/spi/spi-imx.c:213:1: error: expected expression before '}' token
      4.1build error

Signed-off-by: Robin Gong <b38343@freescale.com>
(cherry picked from commit 2e1aff2df420dc5181b6f3f98652e1e044d8ef6c)
---
 drivers/spi/spi-imx.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/spi/spi-imx.c b/drivers/spi/spi-imx.c
index 1e9013a..77b7d70 100644
--- a/drivers/spi/spi-imx.c
+++ b/drivers/spi/spi-imx.c
@@ -208,7 +208,7 @@ static bool spi_imx_can_dma(struct spi_master *master, struct spi_device *spi,
 	struct spi_imx_data *spi_imx = spi_master_get_devdata(master);
 
 	if (spi_imx->dma_is_inited &&
-		(transfer->len > spi_imx_get_fifosize(spi_imx))
+		(transfer->len > spi_imx_get_fifosize(spi_imx)))
 		return true;
 	return false;
 }
-- 
1.8.0

